Jenkinsfile

```
@@ -305,38 +305,8 @@ pipeline {
   }
   post {
     cleanup {
-      // Coverage
-      sh script: 'pip3 install --user coverage', label: "Installing coverage"
-      dir("${BASE_DIR}"){
-        script {
-          if (env.ONLY_DOCS == "false") {
-            def matrixDump = pythonTasksGen.dumpMatrix("-")
-            for(vector in matrixDump) {
-              unstash("coverage-${vector}")
-            }
-            // Windows coverage converge
-            readYaml(file: '.ci/.jenkins_windows.yml')['windows'].each { v ->
-              unstash(
-                name: "coverage-${v.VERSION}-${v.WEBFRAMEWORK}"
-              )
-            }
-            sh('python3 -m coverage combine && python3 -m coverage xml')
-            cobertura coberturaReportFile: 'coverage.xml'
-          }
-        }
-      }
-      // Results
-      script{
-        if (env.ONLY_DOCS == "false") {
-          if(pythonTasksGen?.results){
-            writeJSON(file: 'results.json', json: toJSON(pythonTasksGen.results), pretty: 2)
-            def mapResults = ["${params.agent_integration_test}": pythonTasksGen.results]
-            def processor = new ResultsProcessor()
-            processor.processResults(mapResults)
-            archiveArtifacts allowEmptyArchive: true, artifacts: 'results.json,results.html', defaultExcludes: false
-          }
-        }
-      }
+      convergeCoverage()
```

The condition could be here, but NIT

```suggestion
      whenTrue(env.ONLY_DOCS == "false"){
        convergeCoverage()
      }
```

---

neat!

See https://github.com/elastic/apm-agent-python/pull/821/commits/7b0611d4f2ef1bc155e3d1ce1d8c6870291b990d